src/main/java/apoc/date/Date.java

```diff
@@ -0,0 +1,43 @@
+package apoc.date;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.stream.Stream;
+import apoc.result.LongResult;
+import org.neo4j.procedure.Name;
+import org.neo4j.procedure.Procedure;
+
+
+/**
+ * @author tkroman
+ * @since 9.04.2016
+ */
+public class Date {
+
+	@Procedure
+	public Stream<LongResult> unixtime(@Name("dateProperty") String dateField) {
+		return unixtime_f(dateField, null);
+	}
+
+	@Procedure
+	public Stream<LongResult> unixtime_f(@Name("dateProperty") String dateField, @Name("pattern") String pattern) {
+		if (dateField == null) {
+			return null;
+		}
+		try {
+			return Stream.of(new LongResult(getFormat(pattern).parse(dateField).getTime() / 1000));
+		} catch (ParseException e) {
+			throw new RuntimeException(e);
+		}
+	}
+
+	private static DateFormat getFormat(String pattern) {
+		if (pattern == null) {
+			return new SimpleDateFormat("EEE MMM dd HH:mm:ss zzz yyyy");
```

Comment:
Should we use the ISO8601 time format as default? or sth like `"yyyy-mm-dd HH:MM:SS"` ?